elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/framework/SQLUnitTest.java

```diff
@@ -62,7 +76,31 @@
 
     protected QueryEngine.Transaction transaction;
 
-    public static void init() {
+    // Standard set of test queries used in dialect tests
+    protected enum TestQueryName {
+        WHERE_METRICS_ONLY,
+        WHERE_DIMS_ONLY,
+        WHERE_METRICS_AND_DIMS,
+        WHERE_METRICS_OR_DIMS,
+        WHERE_METRICS_AGGREGATION,
+        HAVING_METRICS_ONLY,
+        HAVING_DIMS_ONLY,
+        HAVING_METRICS_AND_DIMS,
+        HAVING_METRICS_OR_DIMS,
+        PAGINATION_TOTAL,
```

Comment:
What about pagination offset and limit?

Reply:
how should the queries differ for those?

Reply:
There are three things you can ask for pagination:
* totals - total number of rows
* offset - starting row
* limit - number of rows returned in the response.

Each dialect will have a different way of paginating offset and limit.